conjure/errorwriter.go

```diff
@@ -340,6 +342,84 @@ func astErrorParametersMethod(errorDefinition spec.ErrorDefinition, info types.P
 	}
 }
 
+// astErrorSafeParamsMethod generates SafeParams function for an error, for example:
+//
+//  func (e *MyNotFound) SafeParams() map[string]interface{} {
+//  	return map[string]interface{}{"safeArgA": e.safeArgA, "safeArgB": e.safeArgB}
+//  }
+func astErrorSafeParamsMethod(errorDefinition spec.ErrorDefinition, info types.PkgInfo) astgen.ASTDecl {
+	var keyValues []astgen.ASTExpr
+	for _, fieldDefinition := range errorDefinition.SafeArgs {
```

Comment:
should we add errorInstanceId as an extra one here?

Reply:
Yes, we definitely should. Good catch.